R/splitBy.R

```diff
@@ -0,0 +1,181 @@
+#' Split \code{SummarizedExperiment} by grouping variables
+#'
+#' @param x A
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#'   object
+#'
+#' @param grouping A single character value for selecting the grouping variable
+#'   from \code{colData} or \code{rowData}.
+#' 
+#' @param direction A single character value for selecting from where grouping
```

Comment:
This is often called `MARGIN` in the `apply` family of functions. How about using that?